src/types/types.ts

```diff
@@ -250,6 +251,7 @@ export type TasksQuery = {
   from?: number
 }
 export type CancelTasksQuery = Omit<TasksQuery, 'limit' | 'from'> & {}
+export type DeleteTasksQuery = Omit<TasksQuery, 'limit' | 'from'> & {}
```

Comment:
This is the right choice, even if they are "duplicated" structures, because this will allow them to evolve and change separately, awesome!